Dockerfile.rhel

```diff
@@ -6,15 +6,15 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 #
-FROM mikefarah/yq as builder
-RUN apk add --no-cache bash
-COPY .htaccess README.md ./scripts/*.sh /build/
+FROM alpine:3.10 AS builder
```

Comment:
Do we use this file? Why not delete it?

Reply:
I'll create an issue for moving to ubi8, I don't think alpine is necessary, moreso convenient.

The only concern would be due to the difference in image size, since the broker has to be pulled for every workspace start, but I wonder how much of an issue this would actually be considering it would probably be cached pretty quickly.

Reply:
Ah nevermind I see you've beat me to the punch: https://github.com/eclipse/che/issues/14113

I'd do it as a separate PR as I don't think it's a simple drop-in replacement.